projects/admin/src/app/circulation/checkout/checkout.component.ts

```diff
@@ -0,0 +1,252 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { Item, ItemAction } from '../items';
+import { ActivatedRoute, Router } from '@angular/router';
+import { ItemsService } from '../items.service';
+import { DialogService, RecordService } from '@rero/ng-core';
+import { ToastrService } from 'ngx-toastr';
+import { User } from '../../class/user';
+import { UserService } from '../../service/user.service';
+import { TranslateService } from '@ngx-translate/core';
+import { PatronService } from '../../service/patron.service';
+import { map } from 'rxjs/operators';
```

Comment:
Group imports: 

1) Angular + rxjs
2) Third party libraries
3) Local files

This remark is valid for some other files in this commit.